### DISCUSSION TITLE
Add function to manually disconnect/reconnect to servers

### DISCUSSION THREAD

**Discussion by xrgtmx** [OpenRCT2/OpenRCT2#21730] [Feature requests]
Sometimes, when I'm on a server (particularly ones from FFA-Tycoon.com), I can't place objects down or toggle "no entry" to signs, forcing me to leave the server and come back in. Is there a way to add a disconnect/reconnect function under the server icon, to speed up the process?

**Comment by ZehMatt** [comment]
Sounds like a bug that we should fix and not create workarounds. Is there an issue for this? If not please create one.
